towncrier.toml

```diff
@@ -0,0 +1,8 @@
+[tool.towncrier]
```

Comment:
Why is this file checked in? Is it different than the one created by ts_pre_commit_conf?